action/audit.go

```diff
@@ -0,0 +1,41 @@
+package action
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/muesli/crunchy"
+	"github.com/urfave/cli"
+)
+
+// Audit validates passwords against common flaws
+func (s *Action) Audit(c *cli.Context) error {
+	t, err := s.Store.Tree()
+	if err != nil {
+		return err
+	}
+
+	validator := crunchy.NewValidator()
+	var out io.Writer
```

Comment:
I admit it doesn't really serve any purpose here, but I copied the basic skeleton from action/list.go. At this point I wasn't quite sure if you intend to handle output in other ways in the future, so I decided to stick to the basic pattern I found there. By now I think I realized you're just doing that for multi-page outputs action/list.